mod/likes/actions/likes/add.php

```diff
@@ -19,7 +19,7 @@
 }
 
 // limit likes through a plugin hook (to prevent liking your own content for example)
-if (!$entity->canAnnotate(0, 'likes')) {
+if (!_likes_can_like($entity)) {
```

Comment:
this change will have major BC issues, as some plugins now already use this hook to prevent liking... those entities will become likeable again after this change